examples/hallowelt/go_app/README.md

```diff
@@ -0,0 +1,253 @@
+# HelloWorld tutorial using Go client Fabric SDK
+This tutorial shows how to invoke Fabric Private Chaincode using the Go Fabric Client SDK and the Fabric-Private-chaincode(FPC) framework.  This assumes familiarity with the [concepts](https://hyperledger-fabric.readthedocs.io/en/release-2.0/whatis.html#), [the programming model](https://hyperledger-fabric.readthedocs.io/en/release-2.0/chaincode.html) and the [gateway] (https://hyperledger-fabric.readthedocs.io/en/release-2.0/developapps/gateway.html) in Hyperledger Fabric v2.0.
```

Comment:
```suggestion
This tutorial shows how to interact with a FPC Chaincode using the FPC Client SDK for Go.  This assumes familiarity with the [concepts](https://hyperledger-fabric.readthedocs.io/en/release-2.2/whatis.html#), [the programming model](https://hyperledger-fabric.readthedocs.io/en/release-2.2/chaincode.html) and the [gateway](https://hyperledger-fabric.readthedocs.io/en/release-2.2/developapps/gateway.html) in Hyperledger Fabric v2.2.
```

Reply:
In this intro we should also link to our FPC Client SDK docu and give a hint that we have more examples in xyz.

Reply:
ah! good point